models/velodrome/optimism/velodrome_optimism_bribe_mappings.sql

```diff
@@ -0,0 +1,64 @@
+{{
+    config(
+        schema = 'velodrome_optimism',
+        alias='bribe_mappings',
+        materialized = 'incremental',
```

Comment:
if this is incremental, can filters be applied on date?

Reply:
making this a view. It should/could completely re-run on every db update. There could be cases where the bribe and gauge are created at different times.